src/controllers/SubscriberController.ts

```diff
@@ -160,4 +160,22 @@ export class SubscriberController {
         const callBack = this.config.onFontsChanged;
         callBack && callBack(JSON.parse(fonts));
     };
+
+    /**
+     * Listener on selected layout id, if this changes, this listener will get triggered with the updates
+     * @param layoutId the currently selected layout id
+     */
+    onSelectedLayoutIdChanged = (layoutId: string) => {
+        const callBack = this.config.onSelectedLayoutIdChanged;
+        callBack && callBack(layoutId);
+    };
+
+    /**
+     * Listener on layouts, if this changes, this listener will get triggered with the updates
```

Comment:
Would be useful to add a little more detail to the description. E.g. what causes a layout to change, some examples of user interactions that will trigger this. In the future I would like to add some guidance for integrators with 'do's' and 'dont's' in here